types.go

```diff
@@ -16,6 +16,7 @@ type TemporalConfig struct {
 	APIKeys      `json:"api_keys,omitempty"`
 	Endpoints    `json:"endpoints,omitempty"`
 	Orchestrator `json:"orchestrator,omitempty"`
+	LogDir       string `json:"log_dir,omitempty"`
```

Comment:
im pretty sure zap logging path config denotes a *file*, not a *directory*

i recommend naming it `log_path`

https://github.com/RTradeLtd/ipfs-orchestrator/blob/master/config/config.go#L22